### ISSUE TITLE
[a11y] Announce unchecked checkboxes

### ISSUE THREAD

**Issue by rockycodes** [civiform/civiform#12799]
**Describe the bug**
Issue: After selecting "Clear selections" Button, the visual state of the page changes as the checkboxes become unchecked, but there are no announcements that selections were cleared.

**WCAG Success Criterion**
4.1.3 Status Messages

**To Reproduce**
PC/JAWS
1. Navigate to the list of checkbboxes in the group "What can I help you with?"
 2. Press Tab/Shift + Tab to navigate the list of checkboxes.
 3. Select a checkbox with Enter or Space.
 4. Navigate to the "Clear selections" Button.
 5. Select the "Clear selections" Button with Enter or Space.
 
 iPhone/VO
 1. Navigate to the list of checkbboxes in the group "What can I help you with?"
 2. Swipe right/left to navigate the list of checkboxes.
 3. Select a checkbox by performing a double-tap with 1 finger.
 4. Swipe right until "Clear selections" Button.
 5. Select the "Clear selections" Button by performing a double-tap with 1 finger.

**Expected behavior**
 Expectation: After selecting the "Clear selections" Button, there should be an announcement that lets the user know that selections were cleared.

Use an aria-live region or a role="status" to announce something like "Selections cleared" or "All filters removed".
